snapcraft/__init__.py

```diff
@@ -339,6 +339,8 @@ def _get_version():
 
 from snapcraft._baseplugin import BasePlugin  # noqa
 
+from snapcraft._info_file import InfoFile  # noqa
```

Comment:
please put this in internal or keep it here with the _ but do not global import it into the main namespace.

Reply:
if this is generic enough, putting the class in something like file_utils should be good (not sure that is the best place yet)

Reply:
Ah, maybe this could like in the config module (or it could be converted into a package)

Reply:
Changed to be local to remote-build, then we can decide later how to generalize this for remote build and build providers.